rosette/h/BinaryOb.h

```diff
@@ -108,6 +108,7 @@ inline
 Byte&
 ByteVec::byte (int n)
 {
+    // WTH????
```

Comment:
This is an unusual hack. Is it there because old C++ compilers didn't have in-place constructor semantics or something?

Reply:
"Back in the day", it wasn't uncommon to do things like this when you needed an extendible struct.  I don't know for sure, but it seems like this might be what Greg was doing. I sent him an email asking for guidance, but haven't received any reply as yet.

This is somewhat deeply ingrained with his self management of heap data.  It's going to take a bit of work to understand what's going on here unless I can get some input from Greg.

I hadn't intended to check in this commentary at this point, but it slipped through. Dealing with this stuff is the next task I'll be taking on.

Reply:
Understood.